backend/force-ssl.js

```diff
@@ -1,12 +1,8 @@
 const config = require("./config");
 
-module.exports = function(req, res, next) {
-  if (!config.forceHttps) {
+module.exports = function (req, res, next) {
+  if (!config.forceHttps || req.headers["x-forwarded-proto"] === "https") {
     return next();
-  }
-
-  if (req.headers["x-forwarded-proto"] === "https") {
-    next();
   } else {
```

Comment:
I see what you did here - you can as well skip the `} else { }` part. Just put `res.redirect("https://" + req.headers.host + req.url);` without wrapping it with condition statement :)

Reply:
Could you explain? Will it redirect each time, even if the protocol is https? Or does next() function have some special capabilities I'm not aware of?

Reply:
I think I may be wrong here actually

Reply:
I'm not wrong :) You can write it like that (it works the same way):
```js
module.exports = function (req, res, next) {
  if (!config.forceHttps || req.headers["x-forwarded-proto"] === "https") {
    return next();
  }
  res.redirect("https://" + req.headers.host + req.url);
}
```

Reply:
It's because in `if` branch you have a `return` statement which means program flow will not reach the bottom part of this function if the `if` condition is matched. Just less {moustaches} in the code 😉

Reply:
Ok, now I'm convinced! :D